overlord/configstate/config/helpers.go

```diff
@@ -133,6 +135,20 @@ func PatchConfig(snapName string, subkeys []string, pos int, config interface{},
 			config[subkeys[pos]] = result
 			return config, nil
 		}
+
+	case map[string]*json.RawMessage:
```

Comment:
Sorry, sill question, it's so easy to get lost in the logic of this package :( (not your fault). Do we need to handle this explicitly if map[string]interface{} and *json.RawMessage are (recursively) handled by above cases?

Reply:
yes, I'm not sure why we need these changes to helpers, there's no explanation for it in the description of the PR

Reply:
I removed this part too